src/miniflask/miniflask.py

```diff
@@ -177,10 +188,10 @@ def import_module(self, module_name):
 
         # system import (e.g. miniflask internal modules or pip-module repositories)
         if importpath == "system":
-            return import_module(module_spec["importname"])
+            return import_module(module_spec["id"])
```

Comment:
Is there special care needed in case the module directory has been renamed in the mf.init-call?

Reply:
Thanks for clarifying! That would be an interesting way foreward, let's discuss this further in another PR/Issue.